packages/backend/migrate-mongo-config.js

```diff
@@ -0,0 +1,18 @@
+const config = {
+  mongodb: {
+    url:
+      process.env.MONGODB_CONNECTION_STRING ||
```

Comment:
Shall we take those values from node-config-ts?

Reply:
yes, I've found a way to do it